apps/web/src/App.tsx

```diff
@@ -14,6 +14,7 @@ import SignUpPage from './pages/auth/SignUpPage';
 import HomePage from './pages/HomePage';
 import TemplateEditorPage from './pages/templates/editor/TemplateEditorPage';
 import NotificationList from './pages/templates/TemplatesListPage';
+import SubscribersList from './pages/subscribers/SubscribersListPage';
```

Comment:
maybe this component should be also called with `Page` at the end like the others?

Reply:
Can you please explain a bit?